.github/scripts/init_linux_port_vm_network.sh

```diff
@@ -5,37 +5,85 @@
 #
```

Comment:
should we add 
set -o errexit # Exit if command failed. 
set -o nounset # Exit if variable not set. 
set -o pipefail # Exit if pipe failed.


https://github.com/Azure-Samples/iot-middleware-freertos-samples/blob/06d8de660ca1a90e8ca8c2b03eaf4161060decfc/.github/scripts/ci_tests.sh#L6

Reply:
Sounds like a good idea.